### PR TITLE
fix 8034 Integration fails to start: prometheus config cannot be found

### PR DESCRIPTION
**Pull request by KurtStam**
#8034
https://issues.redhat.com/browse/ENTESB-13085

### CODE REVIEW

**Comment by KurtStam** [comment]
Yes I totally agree and I opened a new issue: #8128

**Comment by zregvart** [comment]
Aren't we creating a test that should be (and probably is) in Camel already?

**Comment by KurtStam** [comment]
@zregvart the issue we that we excluded all dependencies. I'm not sure how a test in Camel is going to help. Or maybe you mean something else?

File: app/server/runtime/src/test/java/io/syndesis/server/runtime/integration/ProjectDSLGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        final Document openApiDoc = Library.readDocumentFromJSONString(new String(openApiBytes, StandardCharsets.UTF_8));
+
+        final StringBuilder code = new StringBuilder();
+        RestDslGenerator.toAppendable((OasDocument) openApiDoc)
+            .withClassName("RestRoute")
+            .withPackageName("io.syndesis.example")
+            .withoutSourceCodeTimestamps()
+            .generate(code);
+
+        Assert.assertTrue(code.length() > 0);
```
Comment (zregvart) [inline app/server/runtime/src/test/java/io/syndesis/server/runtime/integration/ProjectDSLGeneratorTest.java]: This test here, doesn't test much about dependencies or anything within Syndesis, right?

That kind of testing is already done in [Camel](https://github.com/apache/camel/tree/master/tooling/openapi-rest-dsl-generator/src/test/java/org/apache/camel/generator/openapi).

**Pull request merged**